src/cascadia/TerminalSettingsEditor/SettingContainer.cpp

```diff
@@ -146,6 +146,30 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
             }
         }
 
+        if (const auto& child{ GetTemplateChild(L"Expander") })
+        {
+            if (const auto& expander{ child.try_as<Microsoft::UI::Xaml::Controls::Expander>() })
+            {
+                // apply header text as name (automation property)
+                if (const auto& header{ Header() })
+                {
+                    const auto headerText{ header.try_as<hstring>() };
+                    if (headerText && !headerText->empty())

```

Comment:
I believe an empty hstring is the same as a null hstring, so you can just put this all in one condition?

```
if (const auto headerText{ header.try_as<hstring>() })
```

(same applies below!)

Reply:
(if you make this change, please apply it in the other places we've done this in this file [or at least this function])